lib/templates.js

```diff
@@ -72,6 +72,19 @@ templates.add('send-feedback', `
     padding: 13px 14px;
   }
 
+  #cancel {
+    margin-left: -3px;
+    border: 1px solid black;
+    background-color: white;
+    color: black;
+  }
+
+  #cancel:hover {
+    color: #D9534F;
+    border-color: #D9534F;
+    background-color: white;
```

Comment:
If we decide to keep the background color of the Close button when the cursor hovers over it then, in my opinion, the `Send Feedback` should have a green background for the same.

Reply:
The reason I went for the black border on the button instead of red is to not emphasizing. When it was red it way too attractive and not helpful. When someone hovers over it it's red text and border.

Ideally, we do have the same design for both buttons but I have seen "Cancel" button have a design that makes is less forthcoming so this should be fine. Regarding change the color of send feedback button, how strongly do you feel about it? I am hesitant to change it because most of the apps that use this (even though I don't think a lot of them do!) expect the purple color and we should just change it unless it absolutely nessacary.

Reply:
Let's leave it like this then; I am sure it's not worth the trouble.

Reply:
Nope in no way is this necessary! Hence this can remain how it is.